docs/guides/05-building/cross-platform.md

```diff
@@ -4,4 +4,76 @@ sidebar_position: 2
 
 # Cross-Platform Compilation
 
```

Comment:
We need an introduction here explaining that cross-platform compilation is not a thing, why that is and that the solution below is the recommended solution.

(Also maybe that building on gitlab/whatever also works, but we haven't focused on it yet)

Reply:
I agree completely. Have made some changes to the file however haven't delved into the why it's not possible, more so that it isn't and the GitHub workflow is mostly a workaround this issue.